Gir.toml

```diff
@@ -64,15 +64,15 @@ generate = [
     "Gio.ThemedIcon",
     "Gio.TlsCertificate",
     "Gio.TlsCertificateFlags",
+    "Gio.FileIOStream",
+    "Gio.FileInputStream",
+    "Gio.FileOutputStream",
```

Comment:
Please insert it in such a way that the sorting is preserved

Reply:
Like this? The list is unordered already, maybe someone wants to order it in general :)

Reply:
`generate` array was sorted alphabetically. I missed that items added after `SimplePermission` was not in order.
Can you resort it?

Reply:
Done, I hope it's in the correct order now :)